test/onnx/pytorch_test_common.py

```diff
@@ -157,6 +158,24 @@ def wrapper(self, *args, **kwargs):
     return skip_dec
 
 
+def skip_unsupported_min_ort_version(min_opset_version: str):
```

Comment:
```suggestion
def skip_unsupported_min_ort_version(reason: str, min_opset_version: str):
```

We should still specify reason

Reply:
opset version was a typo?

Reply:
Done